src/components/Article.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Article.scss';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => (
+  <article className="article">
+    <h1 className="article__title">
+      {title}
+    </h1>
+
+    <span className="article__date">
+      {date}
+    </span>
+
+    <p className="article__text">
+      {text}
+    </p>
```

Comment:
What if we don't get a text? Will we render empty p? Could we don't render p if we don't have text value?